### PR TITLE
feat(multi-library): #CRRE-573 change the conf to allow multi-library

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
Addition of the configuration to manage the multi-library

See README

## Checklist tests
The new configuration is visible in crre/config

## Issue ticket number and link
[CRRE-573](https://jira.support-ent.fr/browse/CRRE-573)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/crre/model/config/ConfigMailModel.java
Comment (BastienBenard) [inline src/main/java/fr/openent/crre/model/config/ConfigMailModel.java:1]: ce n'est plus utilisé

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 * Licence : [AGPL v3](http://www.gnu.org/licenses/agpl.txt)
 * Développeur : CGI
 * Financeur : Région Ile de France
-* Description : Le Catalogue de Ressources Régionales Educatives est un service de gestion des achats des ressources numériques
+* Description : Le Catalogue de Ressources Régionales Educatives est un service de gestion de achats des ressources numériques
```
Comment (Sofianebernoussi) [inline README.md:5]: pq?

**Review by Sofianebernoussi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (flmartineau) [reply README.md:5]: "de achats" ?

File: src/main/java/fr/openent/crre/controllers/OrderRegionController.java [hunk 2]
```diff
@@ -1067,7 +1064,7 @@ private Future<JsonObject> sendMail(HttpServerRequest request, MailAttachment at
         Promise<JsonObject> promise = Promise.promise();
 
         //TODO: #Multi Dynamic email
-        String mail = this.mail.getAddress();
+        String mail = ""; //todo CRRE-576
         String title = "Demande Libraire CRRE";
         String body = "Demande Libraire CRRE ; csv : " + attachment.getName();
```
Comment (flmartineau) [inline src/main/java/fr/openent/crre/controllers/OrderRegionController.java:1069]: Tout ca pourrait pas être stocké en i18n ?

**Review by flmartineau — APPROVED** [review]

File: src/main/java/fr/openent/crre/controllers/OrderRegionController.java (hunk 2 above)
Comment (BastienBenard) [reply src/main/java/fr/openent/crre/controllers/OrderRegionController.java:1069]: ça pourra être fais dans CRRE-576

**Pull request merged**